pkgs/applications/video/rtabmap/default.nix

```diff
@@ -55,5 +55,6 @@ stdenv.mkDerivation rec {
     license = licenses.bsd3;
     maintainers = with maintainers; [ ckie ];
     platforms = with platforms; linux;
+    broken = true; # at 2022-09-23
```

Comment:
Please add a short comment why it got marked broken.

Reply:
Done